src/Magneto/CachedQuery.cs

```diff
@@ -66,8 +67,8 @@ public abstract class AsyncCachedQuery<TContext, TCacheEntryOptions, TResult> :
 		Core.AsyncCachedQuery<TContext, TCacheEntryOptions, TResult>, IAsyncCachedQuery<TContext, TCacheEntryOptions, TResult>
 	{
 		/// <inheritdoc cref="IAsyncCachedQuery{TContext,TCacheEntryOptions,TResult}.ExecuteAsync"/>
-		public virtual Task<TResult> ExecuteAsync(TContext context, IAsyncCacheStore<TCacheEntryOptions> cacheStore, CacheOption cacheOption = CacheOption.Default) =>
-			GetCachedResultAsync(context, cacheStore, cacheOption);
+		public virtual Task<TResult> ExecuteAsync(TContext context, IAsyncCacheStore<TCacheEntryOptions> cacheStore, CancellationToken cancellationToken = default, CacheOption cacheOption = CacheOption.Default) =>

```

Comment:
It's common practice to have `cancellationToken` as the last parameter, is there a good reason not to follow that here?

Reply:
Thanks. I did read that article, and I'd decided to not put the `cancellationToken` parameter last, because at the time I thought that it would be more likely that consumers would want to only pass the `cancellationToken` and not the other optional argument (`cacheOption`). This would have meant that they could avoid naming the argument at the call site. However, upon reading your comment and thinking about it some more, I now think the opposite is true (they'd more likely want to pass `cacheOption` and not `cancellationToken`).